src/utilities.ts

```diff
@@ -363,26 +365,6 @@ async function promptForUserInputString(prompt: string): Promise<string> {
     });
 }
 
-function debounce(func, wait, immediate, ...rest) {
-    let timeout;
-    return function () {
-        const context = this,
-            args = [func, wait, immediate, ...rest];
-        const later = function () {
-            timeout = null;
-            if (!immediate) {
-                func.apply(context, args);
-            }
-        };
-        const callNow = immediate && !timeout;
-        clearTimeout(timeout);
-        timeout = setTimeout(later, wait);
-        if (callNow) {
-            func.apply(context, args);
-        }
-    };
-}
-
```

Comment:
This wasn't used anywhere and was re-binding `this`. We can use ts-debounce or debounce from lodash instead of we want to use this down the road.